apps/backoffice/public/locales/en/common.json

```diff
@@ -332,6 +332,20 @@
         "description": "Use this key for programmatic operations of creating a new service or updating an existing one.",
         "shortcut": "Manage API keys"
       },
+      "new-group-api-key": {
+        "title": "Group's API Key",
+        "description": "Select the group for which you want to generate a group API Key.",
+        "card": {
```

Comment:
Consider to remove "card" level and flattening it

Reply:
[fix](https://github.com/pagopa/io-services-cms/pull/1047/commits/1db82e3f28ad9a3dabb402099f2f006d3175c2e3)